src/plugins/cosmos/hooks/useStakingAction/useStakingAction.tsx

```diff
@@ -37,9 +40,14 @@ export const useStakingAction = () => {
         let result
 
         const { chainSpecific, validator, action, value } = data
-        const memo = validator === SHAPESHIFT_VALIDATOR_ADDRESS ? 'Delegated with ShapeShift' : ''
+        const memo =
+          validator === SHAPESHIFT_COSMOS_VALIDATOR_ADDRESS ||
+          validator === SHAPESHIFT_OSMOSIS_VALIDATOR_ADDRESS
+            ? 'Delegated with ShapeShift'
+            : ''
 
-        if (chainId === KnownChainIds.CosmosMainnet) {
+        // this works because cosmos and osmosis staking interfaces are identical
```

Comment:
Is it the case that the staking interfaces for all cosmos-based assets are the same? If we, we could add a `isCosmosChain` helper or something instead.